src/form.svelte

```diff
@@ -49,7 +72,7 @@
 
 </script>
 <div class="title-row">
-  <h6>Add bookmark</h6>
+  <h6 id="bookmark-title">{formTitle} </h6>
```

Comment:
```suggestion
  <h6>{formTitle}</h6>
```
Seems the ID is not used and can be removed. Let's remove the trailing whitespace.